source/tutorials/qiime2-for-experienced-microbiome-researchers.rst

```diff
@@ -0,0 +1,296 @@
+Various importing methods currently available in QIIME 2 are highlighted in the :doc:`QIIME 2 importing
+tutorial <importing>`.
+
+This step has the potential to be the most confusing part of the QIIME 2 pipeline as there are dozens of import and format types to choose from.
+To see a full list of available import/format types use: ``qiime tools import --show-importable-formats`` and ``qiime tools import --show-importable-types``
+
+If you're importing data that you've generated, you'll likely need to generate a :ref:`manifest file <manifest file>`, which is just a text file that  maps each FASTQ or FASTA file to its sample ID and direction (if applicable).
+
+If you have sequencing data with one of two very specific formats (:ref:`EMP <emp import>` or :ref:`Casava <casava import>`), you can directly import the folder containing your sequencing files with
+the ``--type EMPSingleEndSequences`` or ``--type 'SampleData[PairedEndSequencesWithQuality]'`` flags (or their respective paired-end types).
+Otherwise if you don't have one of these two very specific formats, you'll need to make the manifest file to give ``import`` instructions on what and how to import your files.
+
+Demultiplexing sequences
+~~~~~~~~~~~~~~~~~~~~~~~~
+
+If you have reads from multiple samples in the same file, you'll need to demultiplex your sequences.
+
+If your barcodes have already been removed from the reads and are in a separate file, you can use :doc:`q2-demux <../plugins/available/demux/emp-paired/>` to demultiplex these.
+
+If your barcodes are still in your sequences, you can use functions from the :doc:`cutadapt
+plugin <../plugins/available/cutadapt/index>`.
+The ``cutadapt demux-single`` method looks for barcode sequences at the beginning of your reads (5' end) with a certain error tolerance, removes them, and returns sequence data separated by each sample.
+The QIIME 2 forum has a `tutorial on various functions available in cutadapt <https://forum.qiime2.org/t/demultiplexing-and-trimming-adapters-from-reads-with-q2-cutadapt/2313>`__, including demultiplexing.
+You can learn more about how ``cutadapt`` works under the hood by reading their `documentation <https://cutadapt.readthedocs.io/en/stable/index.html>`__.
+
+Note: Currently ``q2-demux`` and ``q2-cutadapt`` do not support demultiplexing dual-barcoded paired-end sequences, but only can demultiplex with barcodes in the forward reads.
+So for the time being for this type of demultiplexing needs to be done outside of QIIME 2 using other tools, for example
+`bcl2fastq <https://support.illumina.com/sequencing/sequencing_software/bcl2fastq-conversion-software.html>`__.
+
+.. _`merge reads`:
+
+Merging reads
+~~~~~~~~~~~~~~
+
+Whether or not you need to merge reads depends on how you plan to cluster or denoise your sequences into amplicon sequence variants (ASVs) or operational taxonomic units (OTUs).
```

Comment:
Add this? (or blend with the line below)
"If you plan to use deblur or OTU clustering methods next, join your sequences now. If you plan to use dada2 to denoise your sequences, _do not merge_ — dada2 performs read merging automatically after denoising each sequence.